RadixWallet/Clients/UserDefaults+Dependency+Extension/UserDefaults+Dependency+Extension.swift

```diff
@@ -244,3 +244,9 @@ public struct BackupResult: Codable, Sendable {
 		case failure
 	}
 }
+
+extension Profile {
+	public var saveHash: String {
+		"\(header.lastModified.timeIntervalSince1970)-\(header.lastUsedOnDevice.id.uuidString)"
```

Comment:
this is not a hash, will this be hashed?

Reply:
True, it is not a hash anymore, I will rename it